javascript/app/routes.js

```diff
@@ -0,0 +1,3 @@
+import { index } from "@react-router/dev/routes";
+
+export default [index("routes/home.tsx")];
```

Comment:
```suggestion
export default [index("routes/home.jsx")];
```

that should resolve the issue :sweat_smile:

Reply:
haha that's what I get for my copy pasta 🙃

Thank you!